enum.c

```diff
@@ -156,18 +162,28 @@ enum_grep(VALUE obj, VALUE pat)
 }
 
 /*
- *  call-seq:
- *     enum.grep_v(pattern)                  -> array
- *     enum.grep_v(pattern) { |obj| block }  -> array
+ * call-seq:
+ *   grep_v(pattern) -> new_array
+ *   grep_v(pattern) {|element| ... } -> new_array
+ *
+ * Returns a new \Array of objects either selected by the given +pattern+
```

Comment:
The first paragraph should mention the relation to #grep (inverted); there should also be a link to Enumerable#grep. Having the same description as for grep would be very confusing.

Regarding "either ... or ..." see grep.